cli/cmd/environment.go

```diff
@@ -34,13 +34,13 @@ Or just save it to a file
 		username := os.Getenv("FASIT_USERNAME")
 		password := os.Getenv("FASIT_PASSWORD")
 
-		// Fall back to the ident of the logged in user
+		// Fall back to the username of the logged in user
```

Comment:
Is this comment necessary?

Reply:
I think it would be useful for non-unix people reading the code wondering where the USER environment variable comes from

Reply:
Looking at windows I could do the same thing with the USERNAME environment variable

Reply:
my opinion is that we can assume this is basis knowledge, but the behavior still needs to be documented (in README).

When it comes to Windows, maybe we could try adding `if os == windows` before checking the `USERNAME` variant?